### PR TITLE
[TM APPROVED BY RAVE] Adds the Joe Joe ruleset.

### PR DESCRIPTION
**Pull request by Iamgoofball**
## About The Pull Request
dreamseeker_yAZRMSkf8y

## Why It's Good For The Game
https://www.youtube.com/watch?v=h2rdM86M2QI

## Changelog
:cl:
add: Adds the Joe Joe ruleset.
/:cl:

### CODE REVIEW

**Comment by optimumtact** [comment]
You thought

**Comment by Mothblocks** [comment]
theyre called rulesets

**Comment by Fikou** [comment]
image

**Comment by Urumasi** [comment]
Who's Joe?

**Comment by RandomGamer123** [comment]
`restricted_roles = list(
		JOB_CAPTAIN,
		JOB_HEAD_OF_SECURITY,
	)`

are we sure we want sec officers to get holoparas

**Comment by Serxule** [comment]
> `restricted_roles = list( JOB_CAPTAIN, JOB_HEAD_OF_SECURITY, )`
> 
> are we sure we want sec officers to get holoparas

let every role have one, why have the hos, and captain especially, not able to

**Comment by D4C-420** [comment]
seen missing from the possible objectives: 
get out of jail and save your dad,
find all the pieces of god's dead sons corpse to walk, 
protect this dank ass fruit for reasons with the side objective of finding out who you really are, 
use this dank ass fruit to become rich and over throw humanity,
complete your dead best friends dream and kill the universe,
win a horse race so some kid does not die. 
and many more!

**Comment by Donpedrito** [comment]
I kind of wish the ruleset's name wasn't so close to the source material it's inspired by; obviously holoparasites are a huge reference, they don't exactly hide that, but I really do think the little detail of the name makes a big difference in how distracting the reference is from being immersed, hence why they're called holoparasites or power miners and not stands.

I do like the idea of a holoparasite-based gamemode, I think it'd be really cool, although of course without any information on what it actually entails I can't offer my opinion on the gamemode itself one way or the other.

**Comment by Shadowflame909** [comment]
Very bizzare, and awesome!

**Comment by Bawhoppen** [comment]
This PR will be an adventure huh

**Comment by thgvr** [comment]
You should add ゴ ゴ ゴ ゴ sprites with the particle system.

**Review by MrMelbert — CHANGES_REQUESTED** [review]
Goof I swear to christ if you work on another antag without putting their "objectives" in the traitor panel

**Comment by Mothblocks** [comment]
Why are you drafting it it's already DNM

File: code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm [hunk 1]
```diff
@@ -771,3 +771,102 @@
...
+	var/list/guardian_users = list()
+
+/datum/dynamic_ruleset/roundstart/joejoe/pre_execute(population)
+	. = ..()
+	var/amount_of_peeps = 0
+	for(var/i in GLOB.new_player_list)
+		var/mob/dead/new_player/player = i
+		if(player.ready == PLAYER_READY_TO_PLAY && player.mind && player.check_preferences())
+			amount_of_peeps++
+			if(amount_of_peeps % 2 == 0) // every second player becomes a STAND
```
Comment (Mothblocks) [inline code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm:804]: oh my god NO

Comment (tralezab) [reply code/game/gamemodes/dynamic/dynamic_rulesets_roundstart.dm:804]: lmao

**Comment by Shadowflame909** [comment]
This is a stone ocean... Goof you can break free from this stone prison

**Comment by notfrying1pans** [comment]
image

**Pull request closed without merging**